newjersey/navigator.business.nj.gov
# feat: [#188163509] add social media and general SEO to starter kits pages

## Description

We wanted to add a social media preview image to the website for all major websites. 

We realized that we should in addition to this likely add some other SEO characteristics



### Ticket



This pull request resolves [#188163509](https://www.pivotaltracker.com/story/show/188163509).

### Approach



### Steps to Test

If you go into my code and swap out the image URL base with http://localhost:3000/img/team-success.jpg,
Then download this https://chromewebstore.google.com/detail/social-share-preview/ggnikicjfklimmffbkhknndafpdlabib
Then activate it on a starter kits page you can see how the preview should look. 

Beyond that you could inspect and go into the <head> and look at the attributes (while on a starterkits page) and see that the <meta> were what you expected them to be. 



### Notes



## Code author checklist
